config/locales/devise.en.yml

```diff
@@ -8,7 +8,7 @@ en:
       send_paranoid_instructions: "If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes."
     failure:
       already_authenticated: "You are already signed in."
-      inactive: "Your account is not activated yet."
+      inactive: "Your account is suspended."
```

Comment:
I think it may be better to handle this in an `inactive_message` method on the user model, as we'll eventually need to support both suspended and not-activated states.

Reply:
This was just us trying to test if devise would show the suspended message when the user is suspended. All we got was the general "check your email or password" 401 message. We will change this back and try to inactive_method in the User model.